frontend/src/components/AnalyticsCard.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { CardActionArea, createTheme } from '@mui/material';
+
+
+type AppProps = {
+    img: string
+}
+
+export default function AnalyticsCard({ img }: AppProps) {
+    return (
+        <Card>
+            <CardActionArea>
+                <CardMedia
+                    component="img"
+                    image={img}
+                    alt="green iguana"
+                />
+                <CardContent>
+                    <Typography gutterBottom variant="h5" component="div">
+                        "Average 4.28s"
```

Comment:
One unresolved issue with AnalyticsCard is lines 23 to 25, we manually typed in the string "Average 4.28s." Thus unlike the two different analytic graphs, the written analysis under the graphs is the same. This is something we can fix later both in AnalyticsCard and AnalyticsGraphs when it calls the cards.